.github/workflows/pr_push.yml

```diff
@@ -0,0 +1,61 @@
+name: PR/push
+
+on: [push, pull_request]
+
+concurrency:
+    group: ${{ github.head_ref }}-${{ github.workflow }}
+    cancel-in-progress: true
+
+jobs:
+    FastBuild:
+        name: Fast build
+        strategy:
+            matrix:
+                os: [ubuntu-latest, windows-latest]
+                include:
+                    - os: ubuntu-latest
+                      nproc: $(nproc)
+                      pool_scalable: 'ON'
+                      jemalloc: 'ON'
+                    - os: windows-latest
+                      nproc: $Env:NUMBER_OF_PROCESSORS
+                      pool_scalable: 'OFF'
+                      jemalloc: 'OFF'
+        runs-on: ${{matrix.os}}
+
+        steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - name: Install apt packages
+          if: matrix.os == 'ubuntu-latest'
+          run: |
+            sudo apt-get update
+            sudo apt-get install -y cmake libjemalloc-dev libnuma-dev libtbb-dev
+
+        - name: Configure CMake
+          run: >
+            cmake
+            -B ${{github.workspace}}/build
+            -DCMAKE_BUILD_TYPE=Release
+            -DUMF_FORMAT_CODE_STYLE=OFF
```

Comment:
`UMF_FORMAT_CODE_STYLE` should be `ON` because it can fail all `Builds` in the 2nd step

Reply:
@KFilipek let's do it in a separate PR